test/e2e_node/resource_metrics_test.go

```diff
@@ -88,6 +88,16 @@ var _ = framework.KubeDescribe("ResourceMetricsAPI", func() {
 					fmt.Sprintf("%s::%s::%s", f.Namespace.Name, pod0, "busybox-container"): boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
 					fmt.Sprintf("%s::%s::%s", f.Namespace.Name, pod1, "busybox-container"): boundedSample(10*e2evolume.Kb, 80*e2evolume.Mb),
 				}),
+
+				"pod_cpu_usage_seconds_total": gstruct.MatchElements(containerID, gstruct.IgnoreExtras, gstruct.Elements{
+					fmt.Sprintf("%s::%s", f.Namespace.Name, pod0): boundedSample(0, 100),
+					fmt.Sprintf("%s::%s", f.Namespace.Name, pod1): boundedSample(0, 100),
```

Comment:
A non-zero lower bound seems like it would be a better test, to ensure that the metric is actually being populated and not just stuck at zero. Do you know what the granularity of this is? (e.g. could it be set to 1 microsecond?)

I wouldn't block the PR on this, since it's consistent with the existing test though.